backend-api/template.yaml

```diff
@@ -165,6 +165,9 @@ Conditions:
       - !Ref Environment
       - production
 
+  IsDevOrBuild: !Not
+    - !Condition isNotDevOrBuild
```

Comment:
Not sure if it's worth making this more explicit rather than using an existing condition? I think it might also be good to not couple these

Reply:
Please may we not couple these as you suggest

Reply:
Updated